### PR TITLE
Reworked med analyzers

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
image

(ignore the lack of a linebreak on the first line)

Baymed analyzers woo
Scans for:

- Temp
- Brain damage
- general damage
- Genetic damage
- fractures
- tendons
- arteries
- embedded objects
- radiation
- blood levels
- time of death

Advanced health analyzer gives more precise info, removed from station-accessibility, now restricted to admins & syndicate duffel bags (one bag exists in a ruin which i left)

### CODE REVIEW

**Comment by Zonespace27** [comment]
Also touches up wound analyzers while i was nearby

File: code/__DEFINES/time.dm [hunk 1]
```diff
@@ -10,6 +10,9 @@
 ///How many years into the future (or past, if you're into that) the server is
 #define STATION_YEAR_OFFSET 805
 
+/// Convert world time into station time in the hour:minute format
+#define worldtime2stationtime(time) time2text(SSticker.gametime_offset + time, "hh:mm")
```
Comment (Kapu1178) [inline code/__DEFINES/time.dm]: stationtime2text() by default prints a station time stamp in a passed format.

File: code/__HELPERS/maths.dm [hunk 2]
```diff
@@ -261,3 +261,6 @@
 /// Returns the angle of the matrix according to atan2 on the b, a parts
 /matrix/proc/get_angle()
 	return Atan2(b, a)
+
+/// Converts kelvin to farenheit
+#define KTOF(kelvin) (kelvin * 1.8) - 459.67
```
Comment (Kapu1178) [inline code/__HELPERS/maths.dm]: All temperature is by default in kelvin, so you can name this FAHRENHEIT()

File: code/game/objects/items/devices/scanners/health_analyzer.dm [hunk 3]
```diff
@@ -93,302 +88,134 @@
...
+
+		// Limb damage
+		if(verbose)
+			data_string_list += span_bold("Specific limb damage:\n")
+			var/list/damaged_limbs = carbon_target.get_damaged_bodyparts(TRUE, TRUE)
+			if(!length(damaged_limbs))
+				data_string_list += "No detectable limb injuries.\n"
+
+			for(var/obj/item/bodypart/limb as anything in damaged_limbs)
+				var/limb_string = "[capitalize(limb.name)][(limb.bodytype & BODYTYPE_ROBOTIC) ? " <span style='font-weight: bold; color: [COLOR_MEDICAL_ROBOTIC]'>(Cybernetic)</span>" : ""]:"
```
Comment (Kapu1178) [inline code/game/objects/items/devices/scanners/health_analyzer.dm]:
```suggestion
				var/limb_string = "[capitalize(limb.name)][!IS_ORGANIC_LIMB(limb) ? " <span style='font-weight: bold; color: [COLOR_MEDICAL_ROBOTIC]'>(Cybernetic)</span>" : ""]:"
```

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

**Pull request merged**